isotovideo

```diff
@@ -247,7 +247,7 @@ OpenQA::Isotovideo::Utils::checkout_git_repo_and_branch('CASEDIR');
 # multiple distributions or flavors in one repository.
 $bmwqemu::vars{PRODUCTDIR} ||= $bmwqemu::vars{CASEDIR};
 
-$bmwqemu::vars{NEEDLES_DIR} //= $bmwqemu::vars{PRODUCTDIR} . '/needles';
+$bmwqemu::vars{NEEDLES_DIR} //= "$bmwqemu::vars{PRODUCTDIR}/needles/";
```

Comment:
Minor comment: why add extra "/" compare with origin line? 
BTW: When this PR can be merged? I need this PR merge quickly and i can validate some issue on OpenQa env.

Reply:
I removed the extra "/". No usage of that variable seems to expect it.

The only reason this has not been merged is that nobody gave a positive review.